src/main/java/com/google/sps/model/GeocodingClient.java

```diff
@@ -33,26 +35,35 @@
    * @throws GeocodingException A custom exception is thrown to signal an error pertaining to the
    *     Geocoding API.
    */
-  GeocodingResult getGeocodingResult(String address) throws GeocodingException;
+  List<GeocodingResult> getGeocodingResult(String address) throws GeocodingException;
 
   /**
    * Parses for coordinates in a resulting call to the Geocoding API.
    *
    * @param result A GeocodingResult returned from the Geocoding API.
    * @return A LatLng representing coordinates.
    */
-  public static LatLng getCoordinates(GeocodingResult result) {
-    return result.geometry.location;
+  public static LatLng getCoordinates(List<GeocodingResult> results) {
```

Comment:
Add some comments in the javadoc explaining you are only returning the location of the first Street Address result or, if that's not found, the location of the first entry in results.

Reply:
Changed, thanks!

Reply:
`RESOLVED`